saiph/inverse_transform_test.py

```diff
@@ -31,6 +31,13 @@ def test_get_random_weighted_columns(weights: List[float], expected_index: int)
     assert result.values[0] == expected_index
 
 
+def test_normalization() -> None:
+    """Verify that the last modality can be sampled when its cumulated sum is greater than 1."""
```

Comment:
I don't really understand this test

Reply:
The df used in this test is a list of probabilities for hypothetical modalities (3 modalities). We see that the sum of the first 2 is 1, so without normalizing this list, there is no way that the 3rd modality can ever be picked.

That's the regression test: fail without the fix, pass with it.